### ISSUE TITLE
Hydro Thunder

### ISSUE THREAD

**Issue by FreyBentos** [Rinnegatamante/DaedalusX64-vitaGL-Compatibility#121]
# Game summary
- Game name: Hydro Thunder
- Game version: USA

# Daedalus X64 summary
- Version: v0.2

# Issues
- Game starts and runs okay however menu textures are all messed up (see screenshots)
- In game Runs okay around 20-25 fps but it feels faster.
- However there is no audio working on this game yet.
-- I experienced texture glitches on the water surface once when playing however I have been unable to re-create this glitch again so may have been fixed by an emulator restart.

# Screenshots
https://ibb.co/MNnWTtw
https://ibb.co/K67NjBL

# Recommended labels

- In-game (high)
- Slow

**Comment by Rinnegatamante** [comment]
Try to disable texture cache (or update to nightly) to see if it solves some of the textures issues. (v.0.2 with disabled texture cache has sensibly worse textures than nightly)

**Comment by FreyBentos** [comment]
Yeah it works better on the nightly release and on 0.2 if texture caching is disabled. Howvere if bilinear filtering is enabled there is a green outline to textures and hud elements.

**Comment by Rinnegatamante** [comment]
Fullspeed with recent dynarec optimizations. Frame Limit must be on or the game will be too fast.

**Comment by adrianbass360** [comment]
EUROPE VERSION PLAYABLE WITHOUT SOUND. Disable in-game music and SFX. You can use this config and latest daedalusx64 version

[Configs.zip](https://github.com/user-attachments/files/19819960/Configs.zip)
[DaedalusX64 v0.6d (f6ccfb)(02-15-25).zip](https://github.com/user-attachments/files/19819961/DaedalusX64.v0.6d.f6ccfb.02-15-25.zip)
